src/main/java/racingcar/domain/Times.java

```diff
@@ -0,0 +1,22 @@
+package racingcar.domain;
+
+public class Times {
+
+    private final int times;
+    private final int minTimes = 1;
+
+    public Times(int times) {
+        validateTimes(times);
+        this.times = times;
+    }
+
+    private void validateTimes(int times) {
+        if (times < minTimes) {
```

Comment:
아래와 같이 `minTimes`는 인스턴스 변수가 아니라 상수로 추출해야할 것 같아요. 🤔
```suggestion
    private static final int MIN_TIMES_VALUE = 1;
    
    private final int times;

    public Times(int times) {
        validateTimes(times);
        this.times = times;
    }

    private void validateTimes(int times) {
        if (times < MIN_TIMES_VALUE) {
```